src/panel/chart.tsx

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { observer } from 'mobx-react';
+import * as React from 'react';
+import { PureComponent } from 'react';
+import { ResultTableStore } from './resultTableStore';
+import * as d3 from 'd3';
```

Comment:
Not saying anything needs to change, but is there a reason why we didn't use [chart js](https://www.chartjs.org/), or any other simpler plotting library?

Reply:
I didn't know about chart js